internal/lockclient/simple_client_test.go

```diff
@@ -0,0 +1,72 @@
+package lockclient
+
+import (
+	"os"
+	"testing"
+
+	"github.com/GoPlayAndFun/LocKey/internal/lockservice"
+	"github.com/GoPlayAndFun/LocKey/internal/node"
+	"github.com/rs/zerolog"
+)
+
+func TestAcquire(t *testing.T) {
+
+	quit := make(chan bool)
+	go func() {
+		for {
+			select {
+			case <-quit:
+				return
+			default:
+				zerolog.New(os.Stdout).With()
+
+				log := zerolog.New(os.Stdout).With().Logger().Level(zerolog.GlobalLevel())
```

Comment:
Good job on understanding the use cases of the select stmt.
But creating the `log` everytime is not good practice. New variables like `log`,`ls` and `scfg` can be created at the top and used inside the goroutine.

Reply:
These functions have been moved outside of the go routine.